topics/fair/tutorials/data-management-peatlands/tutorial.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: tutorial_hands_on
+title: Introduction to Data Management Plan (DMP) for Peatland Research and PeatDataHub
+level: Introductory
+abbreviations:
+  
+  DMP: Data Management Plan
+  FAIR: Findable, Accessible, Interoperable, Reusable
+zenodo_link: ''
+questions:
+- Why create a DMP for peatland research?
+- How to apply FAIR data management solutions for peatland research?
+Objectives:
+- Understand the importance of data management in peatland research.
+- Learn how to create a data management plan (DMP) for peatland research
+- Create a first draft of a data management plan for peatland research
+time_estimation: "30M"
+key_points:
+- A Data Management Plan (DMP) is a live document that provides a framework for managing data material during and after the research project.
+- FAIR data are data which meet principles of findability, accessibility, interoperability, and reusability (FAIR).
+
+tags:
+- fair
+- open
+- data management plan
+priority: 1
+contributions:
+  authorship:
+    - GabrielaLopez
+    - kkamieniecka
+    - robertmand
+    - angiolini
+    - poterlowicz-lab
+  editing:
+    - hexylena
+subtopic: fair-data
+
+---
+
+
+# Introduction
```

Comment:
We now try to remove this Introduction header, makes it easier to generate informative preview links on e.g. social media
```suggestion

```